docs/installation/mac.md

```diff
@@ -96,17 +97,15 @@ To start elasticsearch as a daemonized process:
 
 ### Installing Elasticsearch with Homebrew
 
-The following directions were
-[taken from the Elasticsearch docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/brew.html).
-
 ```shell
 brew tap elastic/tap
-brew install elastic/tap/elasticsearch-oss
+brew install https://raw.githubusercontent.com/elastic/homebrew-tap/bed8bc6b03213c2c1a7df6e4b9f928e7082fae46/Formula/elasticsearch-oss.rb
```

Comment:
@rhymesIi feel like a lot of people are going to be confused about why we're choosing to do an installation from https://raw.githubusercontent.com/elastic/homebrew-tap/bed8bc6b03213c2c1a7df6e4b9f928e7082fae46/Formula/elasticsearch-oss.rb instead of using the tap available (I am too without context), maybe we can just as a one liner that explains it? Is it for the versioning?

Reply:
Sure, it's because the available one points to ES 7.6 which doesn't work with our gem, I'm going to clarify that

Reply:
updated, thanks!